packages/toolpad-app/src/toolpad/AppEditor/PageEditor/ComponentCatalog/ComponentCatalog.tsx

```diff
@@ -156,7 +166,28 @@ export default function ComponentCatalog({ className }: ComponentCatalogProps) {
                 flexDirection={'row'}
                 justifyContent="space-between"
               >
-                <Typography variant="overline">Custom Components</Typography>
+                <Box display="flex" alignItems="center">
+                  <Typography mr={0.5} variant="overline">
+                    Custom Components
+                  </Typography>
+                  <Tooltip
+                    title={
+                      <Typography variant="inherit">
+                        Expand Toolpad with your own React components.{' '}
+                        <Link
+                          href="https://mui.com/toolpad/concepts/custom-components"
+                          target="_blank"
+                          rel="noopener"
+                        >
+                          Click{' '}
+                        </Link>
+                        to read more.
+                      </Typography>
+                    }
+                  >
+                    <HelpOutlineIcon />
```

Comment:
The icon looks great, my only nitpick is that it contrasts more with the background that other more important visual elements near it. 
Could we make the color slightly more faded, both in light and dark mode? (so not the same exact color as the text)
There are more faded colors in that panel that we should be able reuse in the icon, no need to create new colors.

Reply:
Update the screenshots in description and separated out component.